packages/api/src/Client.ts

```diff
@@ -65,7 +67,7 @@ export default class Client extends EventEmitter {
     super();
 
     this.options = {
-      timeout: 60 * 1000 * 10, // 10 minutes
+      timeout: 90 * 1000, // 90 seconds
```

Comment:
Zlatko suggested that we keep this at 10 minutes, and if necessary, we can look at overriding the timeout on a per-command basis.

I'll revert this change here in the interests of getting a beta build out with this fix.